modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/Helpers/JsonFromTokenUri.swift

```diff
@@ -69,49 +102,72 @@ final class JsonFromTokenUri {
         return .just(.init(tokenId: tokenId, value: json, source: .fallback))
     }
 
-    private func fetchTokenJson(forTokenId tokenId: String, tokenType: TokenType, uri originalUri: URL, address: AlphaWallet.Address, enjinToken: GetEnjinTokenQuery.Data.EnjinToken?) -> AnyPublisher<NonFungibleBalanceAndItsSource<JsonString>, SessionTaskError> {
-        struct Error: Swift.Error {
+    struct Error: Swift.Error {
```

Comment:
This shadows `Swift.Error` and breaks https://github.com/AlphaWallet/alpha-wallet-ios/pull/6285/commits/d3e598fc1ba31522c5f8becb31813376e11a7ac8#diff-e52096e845d342ac700a3a0592f1d0906a26ada51f2e385883b75cd934c1e0e3R175. Maybe give it another name if we want to keep it?

Reply:
ok